src/sparseml/optim/helpers.py

```diff
@@ -136,6 +93,27 @@ def load_recipe_yaml_str_no_classes(recipe_yaml_str: str) -> str:
     return yaml.safe_load(classless_yaml_str)
 
 
+def load_recipe_variables_from_yaml(file_path: Union[str, File]) -> Dict[str, Any]:
```

Comment:
nit: description of what the function does. It seems the intention is to only grab global variables? But that doesn't seem obvious to me from the function name

Reply:
renamed!